packages/experiments/src/components/FolderCover/FolderCover.tsx

```diff
@@ -125,13 +124,19 @@ export interface IFolderCoverLayout {
   contentSize: ISize;
 }
 
-export function getFolderCoverLayout(element: JSX.Element): IFolderCoverLayout {
+export function getFolderCoverLayout(element: JSX.Element, isFluent?: boolean): IFolderCoverLayout {
```

Comment:
It seems really odd to me to see something like this as a flag. I would have expected something like this to come from a theme token or something. @JasonGore

Reply:
this is something we've added to support the new "Fluent FileType Icons", @ThomasMichon and Edgar (dunno his git handle) are in the loop

Reply:
I didn't know about `isFluent` classnaming in general, but to echo Ken's concern, adding component arguments and logic for particular themes won't scale. I'm really concerned about adding the `isFluent` argument to component code.

I have a few questions:
1. Where would the `isFluent` arg come from in code? If at all possible this should be isolated to styling / parent classNaming / CSS.
1. Are there any plans to convert FolderCover to use JS styling?

Not sure if @Jahnp would have any input either.